### ISSUE TITLE
feat: Browser usage docs/example

### ISSUE THREAD

**Issue by DougAnderson444** [candlecorp/wick#461]
**Is your feature request related to a problem? Please describe.**
The problem is I would like to try to run Wick components in the browser, but it's unclear to me if that's possible and how.

**Propose a solution**
Possible solution is a basic example in the examples folder and/or details in the docs on how-to.

**Describe alternatives you've considered**
I did find the `https://www.npmjs.com/package/@candlecorp/wick` package, but it lacks a README or any docs.

**Comment by jsoverson** [comment]
Thanks for the issue @DougAnderson444. I'm working on that now so good timing. I will have more docs and tutorials out this week.

**Comment by DougAnderson444** [comment]
That's awesome @jsoverson! Thanks so much, this Wick project is super cool looking and I'm interested in building some browser components. I've been following the WIT Component Model and have even contributed to `jco` in order to get things working in the browser. Very interested in seeing how it would work with Wick.

**Comment by jsoverson** [comment]
@DougAnderson444 
- I updated the WebAssembly tutorial here: https://candle.dev/docs/wick/getting-started/webassembly/
- I added a component loader [here](https://candle.dev/docs/wick/reference/sdks/browser/), which is also an example of [a minimal Vite/Svelte app](https://github.com/candlecorp/wick-component-loader) here.
- I added a node.js quickstart [here](https://candle.dev/docs/wick/reference/sdks/nodejs/) and [example project](https://github.com/candlecorp/wick-node-example).

The documentation will improve over time, but this is a start.

**Comment by DougAnderson444** [comment]
Thanks @jsoverson (especially for choosing Svelte & Vite over React 😂). I just need enough to get started and check this out in the browser.

I'll give it a shot and can provide some feedback as I experiment

**Comment by DougAnderson444** [comment]
While I'm here, can you confirm if the LLM Llama example also is running 100% on the browser client too?

**Comment by jsoverson** [comment]
Yes, there's no server-side aspect to any of the in-browser demos.

**Comment by DougAnderson444** [comment]
Ran into the first issue: `build.rs` won't build:

**unknown field `operations`**

```sh
error: failed to run custom build command for `jinja v0.1.0`

Caused by:
  process didn't exit successfully: (exit status: 1)
  --- stdout
  cargo:rerun-if-changed=component.wick

  --- stderr
  Error: Could not parse manifest /home/doug/code/rust/components/jinja/component.wick as YAML: unknown field `operations`,
 expected one of `name`, `metadata`, `package`, `host`, `resources`, `import`, `types`, `requires`, `component`, `tests` at
 line unknown, column unknown
```

**Comment by DougAnderson444** [comment]
Found [`just`](https://github.com/casey/just#installation), will make a PR to add a line to the docs

**Comment by DougAnderson444** [comment]
Even when using `just build` I still get 

```
Error: Could not parse manifest /home/doug/code/rust/components/jinja/component.wick as YAML: unknown field `operations`, expected one of `name`
, `metadata`, `package`, `host`, `resources`, `import`, `types`, `requires`, `component`, `tests` at line unknown, column unknown
```

**Comment by DougAnderson444** [comment]
Looks like the `operations` block needs to be indented one more level to `just build`. Pull request fixing docs is there above.
